lab_4_retrieval_w_clustering/main.py

```diff
@@ -316,6 +478,19 @@ def get_clusters_info(self, num_examples: int) -> list[dict[str, int | list[str]
         Returns:
             list[dict[str, int| list[str]]]: List with information about each cluster
         """
+        if not isinstance(num_examples, int) or num_examples <= 0:
+            raise ValueError('Invalid arguments in get_clusters_info')
+
+        clusters_info: list[dict[str, int | list[str]]] = []
```

Comment:
no straight typing, of course

Reply:
if i rm it, mypy raises an error

Reply:
yes, you should fix it with no straight typings

Reply:
sadly, i literraly have no idea how to fix it without straight typing :( someone in our chat had the same problem and asked for help, but to this day they got no reply

Reply:
ok